client/client.go

```diff
@@ -504,6 +508,11 @@ type (
 		// NOTE: Experimental
 		UpdateWorkflowWithOptions(ctx context.Context, request *UpdateWorkflowWithOptionsRequest) (WorkflowUpdateHandle, error)
 
+		// WorkflowUpdateHandleFromRef creates a handle to the referenced update
+		// which can be polled for an outcome.
+		// NOTE: Experimental
+		WorkflowUpdateHandleFromRef(ref WorkflowUpdateRef) WorkflowUpdateHandle
```

Comment:
I think this should be `GetWorkflowUpdateHandle` and should just take workflow information and update ID easily created by the caller (I think requiring the caller to create/use protos directly is a bit much).

Reply:
I'm find with that. @Quinn-With-Two-Ns do you have a feeling here?

Reply:
I'd +1 that

Reply:
Ok, did this. Renamed as suggested and turned the parameter into a 2-field struct